### PR TITLE
Add `pin_test` dbt seed

### PR DESCRIPTION
**Pull request by dfsnow**
This PR moves the `pin_weird` table from S3 to dbt. It renames the table to better align with its use (of holding test cases), and instantiates it using dbt's seed functionality.

Related to #380.

### CODE REVIEW

File: dbt/seeds/ccao/ccao.pin_test.csv
Comment (dfsnow) [inline dbt/seeds/ccao/ccao.pin_test.csv:1]: Curious what other fields we might add to make this useful. The intention of this table is to be a repository of test PINs for use with integration testing (mainly for the model and apps).

File: dbt/seeds/ccao/docs.md [hunk 1]
```diff
@@ -7,3 +7,25 @@ regressions and reporting classes.
...
+The possible values for `test_type` include:
+
+- `class_change`
+- `incorrect_char`
+- `just_weird`
+- `multi_card`
+- `multi_card_prorated`
+- `omitted_assessment`
+- `prorated`
+- `split_class`
```
Comment (dfsnow) [inline dbt/seeds/ccao/docs.md:28]: Not an exhaustive list, just figured it'd be a good starting place.

File: dbt/seeds/ccao/ccao.pin_test.csv
Comment (dfsnow) [reply dbt/seeds/ccao/ccao.pin_test.csv:1]: Also if anyone has any PIN's they'd like to add then now's the time. I want this to be a running list the we dump things into regularly.

**Review by wrridgeway — APPROVED** [review]

**Review by jeancochrane — APPROVED** [review]
Looks great! Are there any automated processes that read from the existing `pin_weird` table such that we'll need to update them to reference `pin_test` after deploying this change?

**Comment by dfsnow** [comment]
> Looks great! Are there any automated processes that read from the existing `pin_weird` table such that we'll need to update them to reference `pin_test` after deploying this change?

Not currently, but I'm fixin to make some.

**Pull request merged**